AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -1937,7 +1937,7 @@ - (void)testIsNewerVersionFunction {
   XCTAssertFalse(result);
 }
 
-- (void)testStartUpdateWhenApplicationEnterForeground {
+- (void)testStartUpdateWhenApplicatsionDidBecomeActive {
```

Comment:
Extra symbol 's' in the word 'application'

Reply:
Oops.